cmd/node/multi_node.go

```diff
@@ -251,7 +259,9 @@ func StartMultiNode(numOfinstances, layerAvgSize int, runTillLayer uint32, dbPat
 		log.Error("cannot parse genesis time %v", err)
 	}
 	pubsubAddr := "tcp://localhost:55666"
-	events.InitializeEventReporter(pubsubAddr)
+	if err := events.InitializeEventReporter(pubsubAddr); err != nil {
```

Comment:
make sure you close this somewhere when the node is done

Reply:
good catch, fixed